pallets/loans/src/rate_model.rs

```diff
@@ -259,11 +320,26 @@ mod tests {
         let borrow_rate = Rate::saturating_from_rational(2, 100);
         let util = Ratio::from_percent(50);
         let reserve_factor = Ratio::zero();
-        let supply_rate = InterestRateModel::get_supply_rate(borrow_rate, util, reserve_factor);
+        let supply_rate = InterestRateModels::get_supply_rate(borrow_rate, util, reserve_factor);
         assert_eq!(
             supply_rate,
             borrow_rate
                 .saturating_mul(((Ratio::one().saturating_sub(reserve_factor)) * util).into()),
         );
     }
+
+    #[test]
+    fn new_curve_name_correctly_calculates_borrow_rate() {
+        let new_curve_name_model = NewCurveNameModel::new(
+            Rate::from_inner(Rate::DIV / 100 * 110), // 110%
+            Ratio::from_percent(50),
+            Ratio::from_percent(14),
+        );
+        assert_eq!(
+            new_curve_name_model
+                .get_borrow_rate(Ratio::from_percent(10))
```

Comment:
These method parameters were gathered out of the blue and don't have actual or reasonable values. They were used to simply verify that the calculation is correct.